### ISSUE TITLE
[rework] rework w/ paper list

### ISSUE THREAD

**Issue by kozistr** [kozistr/Awesome-GANs#15]
* Select some of the papers which have impacts and important as i think,
* Summary all of the papers within 3 lines!
* Grouping the papers w/ following characteristic

**Comment by DonaldTsang** [comment]
Agreed, but the problem is that more and more papers are coming out, especially with the recorrection with https://github.com/nightrome/really-awesome-gan
Is it possible to judge a paper based on the amount of times it get referenced by other papers (assume metrics by SemanticScholar and ResearchGate)?

**Comment by kozistr** [comment]
> Agreed, but the problem is that more and more papers are coming out, especially with the recorrection with https://github.com/nightrome/really-awesome-gan
> Is it possible to judge a paper based on the amount of times it get referenced by other papers (assume metrics by SemanticScholar and ResearchGate)?

reference count usually is a good metric for judging the paper, but in case of purpose of this repo, is not as i think! 
so, not recommending all of GAN papers, but the paper what i and others think it is good to read ( based on my and others experiences).

**Comment by DonaldTsang** [comment]
In that case, besides popularity, there should be a "uniqueness" factor, since many GANs can be abstracted away to having the same meta-structure. See https://github.com/kozistr/Awesome-GANs/issues/18 for some examples.
Of course I would hope that more varieties of meta-structures would lead to more innovation, but then again that is just hope if implementing all GANs are impossible.
